pkg/sdk/v1beta1/kafkacluster_types.go

```diff
@@ -226,7 +236,7 @@ type AlertManagerConfig struct {
 type ExternalListenerConfig struct {
 	CommonListenerSpec   `json:",inline"`
 	ExternalStartingPort int32  `json:"externalStartingPort"`
-	HostnameOverride     string `json:"hostnameOverride,omitempty"`
+	Hostname             string `json:"hostnameOverride,omitempty"`
```

Comment:
do we need this change?

Reply:
For me, it makes more sense to name the filed `Hostname` and not `HostnameOverride` now since it has a broader role. It was used only to override the hostname value from the CR. Now, it always set, through the `reconcileHostnameForExternalListeners`. And it's either the override value, either the LB value.

Reply:
Might cause some confusion where the Property is `Hostname` and `json:"hostnameOverride,omitempty"`